client/src/components/DataFiles/DataFilesProjectMembers/_cells/ProjectRoleSelector.jsx

```diff
@@ -71,12 +71,13 @@ const ProjectRoleSelector = ({ projectId, username }) => {
       </DropdownSelector>
       {data.role !== selectedRole && !isFetching && (
         <Button
-          className="data-files-btn"
+          type="primary"
           onClick={() =>
             setProjectRole({ oldRole: data.role, newRole: selectedRole })
           }
+          isLoading={isMutating}
         >
-          {isMutating ? <LoadingSpinner placement="inline" /> : 'Update'}
+          Update
```

Comment:
the size and alignment of button seems off from main/prod.
![Screen Shot 2022-08-16 at 7 36 37 PM](https://user-images.githubusercontent.com/8287580/185008592-8f7f0fbd-0d95-45fe-9033-b1cff9fb23b1.png)
![Screen Shot 2022-08-16 at 7 36 32 PM](https://user-images.githubusercontent.com/8287580/185008596-2c2c2649-4905-47d3-a712-bb14e978038c.png)

Reply:
I've never been able to test Shared Workspaces project roles locally. Did this get fixed?

Reply:
yes, the update button is the same larger size as the role button.